packages/core/src/codewhisperer/tracker/lineTracker.ts

```diff
@@ -0,0 +1,166 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
can you add some documentation about what this class is and what its intended to be used for?

Reply:
sgtm, Max fyi I merged #4534 #4535 into this one.